docs/website/docs/dlt-ecosystem/verified-sources/openapi-generator.md

```diff
@@ -0,0 +1,198 @@
+---
+title: OpenAPI source generator
+description: dlt verified source for Personio API
+keywords: [personio api, personio verified source, personio]
+---
+import Header from './_source-info-header.md';
+
+# OpenAPI source generator
+
+<Header/>
+
+Our OpenAPI source generator - `dlt-init-openapi` - generates [`dlt`](https://dlthub.com/docs) data pipelines from OpenAPI 3.x specs using the [dlt rest_api verified source](https://dlthub.com/docs/dlt-ecosystem/verified-sources/rest_api) to extract data from any REST API. If you are not familiar with the `rest_api` source, please read [dlt rest_api](https://dlthub.com/docs/dlt-ecosystem/verified-sources/rest_api) to learn how our `rest_api` source works.
```

Comment:
```suggestion
Our OpenAPI source generator - `dlt-init-openapi` - generates [`dlt`](https://dlthub.com/docs) data pipelines from [OpenAPI 3.x specs](https://swagger.io/specification/) using the [rest_api verified source](https://dlthub.com/docs/dlt-ecosystem/verified-sources/rest_api) to extract data from any REST API. If you are not familiar with the `rest_api` source, please read [dlt rest_api](https://dlthub.com/docs/dlt-ecosystem/verified-sources/rest_api) to learn how our `rest_api` source works.
```

Suggest linking the specs, and removing one "dlt" mention: rest_api is already specific enough

Reply:
done